src/components/assistant-thinking.tsx

```diff
@@ -0,0 +1,15 @@
+export function MessageThinking() {
+	return (
+		<div className="flex justify-center items-center gap-1 p-0.5">
+			<div
+				className="animate-pulse h-1.5 w-1.5 bg-a8c-blueberry rounded-full"
+				style={ { animationDelay: '0.2s' } }
+			/>
+			<div className="animate-pulse h-1.5 w-1.5 bg-a8c-blueberry rounded-full" />
+			<div
+				className="animate-pulse h-1.5 w-1.5 bg-a8c-blueberry rounded-full"
+				style={ { animationDelay: '0.4s' } }
+			/>
+		</div>
+	);
+}
```

Comment:
At some point I think we may want to consider grouping Assistant-related components into their own directory (or rename this component to something more generic). I think this is fine to merge as-is until we're ready for that change, however.